src/lib/elements/typography/typography.utils.ts

```diff
@@ -0,0 +1,38 @@
+import { lineHeights } from './../../theme/design-system/line-heights';
+import {
+  ISizeStyle,
+  TTypographySizes,
+  TTypographyVariants,
+} from './typography.types';
+
+export const getSizeStyle = (
+  variant: TTypographyVariants,
+  size: TTypographySizes
+): ISizeStyle =>
+  ({
+    display: {
+      large: { fontSize: '8xl', lineHeight: lineHeights['6xl'] },
```

Comment:
try access to object the object fontSize `fontSize['8xl']` instead put a string Constant `'8xl`